src/telemetry/telemetry.c

```diff
@@ -101,19 +106,42 @@ bgw_job_type_counts()
 		if (namestrcmp(&job->fd.proc_schema, INTERNAL_SCHEMA_NAME) == 0)
 		{
 			if (namestrcmp(&job->fd.proc_name, "policy_refresh_continuous_aggregate") == 0)
-				counts.policy_cagg++;
+			{
+				if (job->fd.fixed_schedule)
+					counts.policy_cagg_fixed++;
+				else
+					counts.policy_cagg++;
+			}
 			else if (namestrcmp(&job->fd.proc_name, "policy_compression") == 0)
-				counts.policy_compression++;
+			{
+				if (job->fd.fixed_schedule)
+					counts.policy_compression_fixed++;
+				else
+					counts.policy_compression++;
```

Comment:
A lot of stuff is not covered, but I see that you have these jobs in the test... maybe this is a stale comment by codecov bot.